README.md

```diff
@@ -37,20 +37,38 @@ pipenv shell
 
 Now, a python REPL can be started to use the library interactively.
 
+## Running tests ##
+
+Tests can be run, from the project root, using:
+
+```bash
+pytest --doctest-glob='*.rst' docs/
+pytest -x -n 3 --doctest-modules --cov-report term --cov=sem/ ./tests
+```
+
+These two commands will run, respectively, all code contained in the `docs/`
+folder and all tests, also measuring coverage and outputting it to the terminal.
+
+Since we are mainly testing integration with ns-3, tests require frequent
+copy+paste of folders, ns-3 compilations and simulation running. Because of
+this, full tests may need up to 30 minutes to complete. Single test files can be
```

Comment:
How do you get this 30min data? List the hw/sw enviroment you use so people can take it as a reference.